x/autopilot/module_ibc.go

```diff
@@ -133,31 +136,27 @@ func (im IBCModule) OnRecvPacket(
 
 	// Error any transactions with a Memo or Receiver field are greater than the max characters
 	if len(tokenPacketData.Memo) > MaxMemoCharLength {
-		return channeltypes.NewErrorAcknowledgement(errorsmod.Wrapf(types.ErrInvalidMemoSize, "memo length: %d", len(tokenPacketData.Memo)))
+		return channeltypes.NewErrorAcknowledgement(errorsmod.Wrapf(types.ErrInvalidMemoLength, "memo length: %d", len(tokenPacketData.Memo)))
 	}
-	if len(tokenPacketData.Receiver) > MaxMemoCharLength {
-		return channeltypes.NewErrorAcknowledgement(errorsmod.Wrapf(types.ErrInvalidMemoSize, "receiver length: %d", len(tokenPacketData.Receiver)))
+	if len(tokenPacketData.Receiver) > MaxReceiverCharLength {
+		return channeltypes.NewErrorAcknowledgement(errorsmod.Wrapf(types.ErrInvalidReceiverLength, "receiver length: %d", len(tokenPacketData.Receiver)))
 	}
 
-	// ibc-go v5 has a Memo field that can store forwarding info
-	// For older version of ibc-go, the data must be stored in the receiver field
-	var metadata string
```

Comment:
did we check the ibc versions on the main chains?

Reply:
yeah I have [this](https://docs.google.com/spreadsheets/d/18I813HXRCrz3IHq3F4dTAnIe7DaXCxYQWtPoJrCoSdI/edit?gid=0#gid=0) doc where I was tracking the versions to see when we can use the ibc version of ICQ

Only chains I don't know are injective and saga (since they're closed source), but I'm pretty confident they're past SDK v5

Reply:
just confirmed saga is v7 (from chain registry)

Reply:
Nice, thanks for checking